tests/system/providers/amazon/aws/example_dynamodb_to_s3.py

```diff
@@ -195,7 +225,8 @@ def delete_dynamodb_table(table_name: str):
         backup_db_segment_1,
         backup_db_segment_2,
         export_time,
-        backup_db_to_point_in_time,
+        backup_db_to_point_in_time_full_export,
```

Comment:
Insert `latest_export_time` between these two lines

```
       export_time,
       latest_export_time
       backup_db_to_point_in_time_full_export,
```
Without it listed in the chain(), it fires off at the very beginning and causes the test to fail because env_id isn't populated yet.

Reply:
fixed in https://github.com/apache/airflow/pull/41517